packages/app-bridge/src/react/usePortalNavigation.spec.ts

```diff
@@ -0,0 +1,278 @@
+// @vitest-environment happy-dom
+
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { afterEach, describe, expect, it, vi } from 'vitest';
+import { renderHook, waitFor } from '@testing-library/react';
+import { getAppBridgeThemeStub } from '../tests';
+import { usePortalNavigation } from './usePortalNavigation';
+import { NavigationTreeDummy } from '../tests/NavigationTreeDummy';
+
+const DEFAULT_NAVIGATION_TREE = NavigationTreeDummy.default();
+
+describe('usePortalNavigation', () => {
+    afterEach(() => {
+        vi.resetAllMocks();
```

Comment:
Unless I'm missing something, there are no mocks that should be reset - each test case initialises its own mocks.

Reply:
yup... you are totally right